openapi/endpoints/accounts/accounts-get.yaml

```diff
@@ -0,0 +1,21 @@
+get:
+  tags:
+    - accounts
+  summary: Get an account
+  description: Get an Account by account id 
+  parameters:
+  - name: accountId
+    in: path
+    description: Id of the account.
+    example: 225d85e65495722bf6517ea0ba0d6f56
+    required: true
+    schema:
+      type: string
+  responses:
+    "200":
```

Comment:
what about 403? would be good to add for consistency with other endpoints

Reply:
Was discussing this with @ncjones . He suggested that we shouldn’t document common exceptions like 403 resource not found / 403 not authorised , since these apply to all our endpoints …

but since other endpoints have this , maybe it is good to be consistent  🤔.

Reply:
i think it's good to be consistent. if openapi had something like default response codes, would be great, but considering it's not, let's add it dilignently and consistently on every endpoint we have

Reply:
@rb455  finally got a chance to add 403 to this endpoint